examples/java/src/main/java/org/apache/beam/examples/complete/kafkatopubsub/KafkaToPubsub.java

```diff
@@ -151,17 +152,19 @@ public static void main(String[] args) {
     KafkaToPubsubOptions options =
         PipelineOptionsFactory.fromArgs(args).withValidation().as(KafkaToPubsubOptions.class);
 
-    run(options);
+    // Create the pipeline
+    Pipeline pipeline = Pipeline.create(options);
+    run(pipeline, options);
   }
 
   /**
    * Runs a pipeline which reads message from Kafka and writes it to GCS.
    *
    * @param options arguments to the pipeline
    */
-  public static PipelineResult run(KafkaToPubsubOptions options) {
+  public static PipelineResult run(Pipeline pipeline, KafkaToPubsubOptions options) {
     // Configure Kafka consumer properties
-    Map<String, Object> kafkaConfig = new HashMap<>();
+    Map<String, Object> kafkaConfig = parseKafkaConsumerConfig(options.getKafkaConsumerConfig());
```

Comment:
I think it is better to leave the initialization of `kafkaConfig` with an empty `HashMap` and then update it with the result of `parseKafkaConsumerConfig` function. Such explicit initialization may eliminate the necessity to immediately go to the function and see what's going on in it.

Reply:
Fixed